governance/program/src/entrypoint.rs

```diff
@@ -3,10 +3,13 @@
 
 use crate::{error::GovernanceError, processor};
 use solana_program::{
-    account_info::AccountInfo, entrypoint, entrypoint::ProgramResult,
+    account_info::AccountInfo, declare_id, entrypoint, entrypoint::ProgramResult,
     program_error::PrintProgramError, pubkey::Pubkey,
 };
 
+// Todo: Feature-gate this to only run when publishing shank IDL
+declare_id!("GovER5Lthms3bLBqWub97yVrMmEogzX7xNjdXpPPCVZw");
```

Comment:
We may also extend shank to allow you to supply the program ID manually via the command line.
I just assumed people always wanna expose this id via `declare_id`, but it seems that assumption is not correct.

Reply:
`programId` can't be hardcoded here because there are multiple instances of `spl-governance` deployed and hence the generated IDL/SDK should take `programId` as input param and not hardcore it.